documentation-site/pages/components/textarea.mdx

```diff
@@ -21,10 +21,15 @@ import Overrides from '../../components/overrides';
 import {StatefulTextarea as Textarea, SIZE} from 'baseui/textarea';
 import * as TextareaExports from 'baseui/textarea';
 
+import Yard from '../../components/yard/index';
+import textareaYardConfig from '../../components/yard/config/textarea';
+
 export default Layout;
 
 # Textarea
 
+<Yard componentName="Textarea" placeholderHeight={48} {...textareaYardConfig} />
```

Comment:
Should we always remove generated API and overrides inspector when adding yard?

Reply:
yeah, that makes sense